### ISSUE TITLE
Only provide BAM output

### ISSUE THREAD

**Issue by armintoepfer** [cbg-ethz/haploclique#19]
Drop all output files except for BAM. Annotate metrics to each record using custom BAM tags, such as super-read coverage.

**Comment by tobiasmarschall** [comment]
I agree to make BAM the default output. Having some optional "side output" such as FASTA might still be helpful.

**Comment by armintoepfer** [comment]
In my opinion, we can rather provide samtools one-liners to extract certain information in a different format.

**Comment by tobiasmarschall** [comment]
Also makes a lot of sense. That is, if we generated the according documentation.
